src/pages/build-apps/guides/transaction-signing.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Transaction signing
+description: Prompt users to sign and broadcast transactions to the Stacks blockchain
+experience: beginners
+tags:
+  - tutorial
+images:
+  large: /images/transaction-signing.svg
+  sm: /images/transaction-signing.svg
+---
+
+## Introduction
+
+This guide explains how to prompt users to sign [transactions](/understand-stacks/transactions) and broadcast them to the Stacks blockchain by implementing the `connect` package of [Stacks.js](https://blockstack.github.io/stacks.js/).
+
+Transaction signing provides a way for users execute [Clarity smart contracts](/write-smart-contracts/overview) that are relevant to your app then handle the result immediately.
```

Comment:
imho "relevant to your app then handle the result immediately" makes this more confusing. It could be any contract. for contract calls, the results actually aren't returned if I understand it correctly (plus it doesn't respond immediately)